config/features.yml

```diff
@@ -730,7 +734,7 @@ features:
     actor_type: user
     description: Toggle to control the amount of eligible users entering the send_email portion of the flow.
   pdf_warning_banner:
-    description: "When enabled, will allow display of PDF cert warnings in find-a-form"
+    description: 'When enabled, will allow display of PDF cert warnings in find-a-form'
```

Comment:
I'm guessing prettier or something changed these double quotes to single quotes. But maybe just remove them altogether because they don't seem like they're required?

Reply:
Removed these